_config.yml

```diff
@@ -55,6 +54,9 @@ collections:
 
 include:
  - CNAME
+ 
+ github:

```

Comment:
Probably worth a comment as to why we're doing this. E.g. a link to https://github.com/redhat-developer/app-services-architecture/pull/11 would be enough, I think.